src/libraries/HooksLib.sol

```diff
@@ -53,7 +57,7 @@ library HooksLib {
     }
 
     /// @notice Handles the afterVerifySignature hook
-    /// @dev Expected to revert if the signature is invalid
+    /// @notice MUST revert if the signature is invalid
```

Comment:
this is not really true, if the signature is invalid we wouldn't get to the hook callsite no?